buildroot-external/board/pc/generic-x86-64/kernel.config

```diff
@@ -29,6 +32,79 @@ CONFIG_MMC_SDHCI_ACPI=y
 CONFIG_MMC_SDHCI_PCI=y
 CONFIG_X86_INTEL_LPSS=y
 
+# NICs
```

Comment:
Why are you adding those to the board specific config?

The ones which are PCIe specific should be added to `device-support-pcie.config`. This is included for both, ova and generic-x86-64.

Reply:
@angers Those NICs are bounded to x86 platforms in Kconfig, Debian only have them enabled in x86 config as well. I tought we can share `device-support-pcie.config` between x86 and aarch64-generic later on?

Reply:
Ok if that is the case, fair enough.